js/index.js

```diff
@@ -34,3 +35,9 @@ $navLinks.forEach(($navLink) => $navLink.addEventListener('click', handleNavLink
 
 const $logo = document.querySelector('#logo');
 $logo?.addEventListener('click', handleLogoClick);
+
+const $orderOpenButton = document.querySelector('#order-open');
+$orderOpenButton?.addEventListener('click', () => toggleModal());
+
+const $orderCloseButton = document.querySelector('#order-close');
+$orderCloseButton?.addEventListener('click', () => toggleModal());
```

Comment:
```suggestion
const $orderCloseButton = document.querySelector('#order-close');
$orderCloseButton?.addEventListener('click', handleOrderCloseClick);
```

А тут подразумевается только закрытие
Можно создать вместо toggleModal 2 функции обработчика в одной только открытие, в другой только закрытие